pkgs/development/python-modules/pyjson5/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv
+, lib
+, python
+, setuptools
+, buildPythonPackage
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "pyjson5";
+  version = "0.8.5";
+
+  src = fetchFromGitHub {
+    owner = "dpranke";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0nyngj18jlkgvm1177lc3cj47wm4yh3dqigygvcvw7xkyryafsqn";
+  };
+
+  doCheck = false;
+
+  configurePhase = ''
```

Comment:
Please don't remove the default configure phase. This is not necessary with `buildPythonPackage`.